src/slack/error.js

```diff
@@ -0,0 +1,69 @@
+require("dotenv").config();
+const config = require("../config");
+const { getSection, bot, token } = require(".");
+
+const channel = config.SLACK_ALERT_CHANNEL_ID;
+
+let prevErrorMessage = "";
+let prevStackTrace = "";
+let threadTs = "";
+
+// For use by SlackErrorTranport in winston
+const sendAlert = async (error) => {
+  // For new errors (i.e., not the same error as the previous one)
```

Comment:
Can you make it exit the function if `SLACK_ALERT_CHANNEL_ID` is not provided?

Reply:
Yep! Will revise today.

Reply:
This refinement is included in the commits below. Lmk if it's what you had in mind. Thanks for bringing this up. Something was bugging me yesterday and I couldn't quite see it, but this was it.

Reply:
Yep, looks good. Thanks for the update!